src/index.js

```diff
@@ -10,6 +10,7 @@ if (get(window, versionField, null)) {
 
 set(window, versionField, 'v<?= PACKAGE(version) ?>');
 
+export * from './bidders';
```

Comment:
It will make the ad-products on N&S much bigger. Can we build separate package for mobile-wiki?
Quote from ticket:
```
... build as a separate package (bidders.js, bidders.amd.js, bidders.global.js and concat to global-bundle.js)
```
```

Reply:
Right. Separated to bidders.js package.